Makefile

```diff
@@ -143,14 +143,14 @@ distclean: clean
 install: install-base install-mount.mergerfs install-man
 
 install-base: $(TARGET)
-	$(INSTALL) -v -m 0755 -D "$(TARGET)" "$(INSTALLBINDIR)/$(TARGET)"
+	$(INSTALL) -v -m 0755 "$(TARGET)" "$(INSTALLBINDIR)/$(TARGET)"
```

Comment:
The command as it stood made `install` print its usage info and quit. With GNU install, `-D` signifies to create intermediate directories. On FreeBSD it has a totally different meaning:

```
     -D destdir
             Specify the DESTDIR (top of the file hierarchy) that the items
             are installed in to.  If -M metalog is in use, a leading string
             of ``destdir'' will be removed from the file names logged to the
             metalog.  This option does not affect where the actual files are
             installed.
```

I didn't spot an equivalent option. `-d` sounds like it does that from the description, but complains if the source file is not a directory.

`-v` and `-m` behave the same way as far as I can see.